TBAnnotationClustering-Swift/ViewController.swift

```diff
@@ -43,7 +43,7 @@ class ViewController: UIViewController, MKMapViewDelegate {
         let toRemove = NSMutableSet(set: before)
         toRemove.minusSet(after as Set<NSObject>)
         
-        dispatch_async(dispatch_get_main_queue()) { () -> Void in
+        NSOperationQueue().addOperationWithBlock() {
```

Comment:
I'm sorry, you are right.

However, `NSOperationQueue.mainQueue().addOperationWithBlock()` is essentially the same as `dispatch_async(dispatch_get_main_queue())`. I personally prefer dispatch queues and in Swift 3 libdispatch got a really nice "swifty" interface, see https://github.com/apple/swift-evolution/blob/master/proposals/0088-libdispatch-for-swift3.md